src/test/java/org/folio/service/mapping/TranslationFunctionUnitTest.java

```diff
@@ -76,6 +77,14 @@ static void setUp() {
     return Collections.singletonMap(identifierType.getString("id"), identifierType);
   }
 
+  private static Map<String, JsonObject> getInstanceFormats() {
+    JsonObject identifierType =
+      new JsonObject(TestUtil.readFileContentFromResources("mockData/inventory/get_instance_formats_response.json"))
```

Comment:
Do you want to move this fileName to the TestUtil, seems to being referred from multiple classes